doc/radosgw/s3/bucketops.rst

```diff
@@ -14,8 +14,13 @@ Constraints
 In general, bucket names should follow domain name constraints.
 
 - Bucket names must be unique.
-- Bucket names must begin and end with a lowercase letter.
-- Bucket names may contain a dash (-).
+- Bucket names cannot be formatted as IP address.
+- Bucket names can be between 3 and 63 characters long.
+- Bucket names can contain only lower-case characters, numbers, periods, and dashes. 
+- Bucket names must start with a lowercase letter or number.
+- Bucket names cannot contain underscores (_), end with a dash(-), have consecutive periods(.), or use dashes adjacent to periods.
+
+.. note:: The above constrainsts are relaxed if the option 'rgw_relaxed_s3_bucket_names' is set to true except that the bucket names must still be unique, cannot be formatted as IP address and can contain letters, numbers, periods, dashes and underscores for upto 255 characters long.
```

Comment:
constrainsts -> constraints

upto -> up to

Reply:
done